rust-runtime/aws-smithy-http-server-typescript/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+pub mod server;
+pub mod socket;
```

Comment:
Both modules here are not used because of the issue with napi not being able to re-export from another crate. 

I would completely get rid of this crate for now, since it is unused and rely on code generation until there is a solution where we can abstract this code into a runtime crate.